core/src/main/scala/org/apache/spark/rpc/netty/NettyStreamManager.scala

```diff
@@ -65,6 +65,10 @@ private[netty] class NettyStreamManager(rpcEnv: NettyRpcEnv)
     }
   }
 
+  override def removeFile(file: File): Unit = {
+    files.remove(file.getName)
```

Comment:
It looks like making 'File' as argument is redundant. File path is all which is needed. Please check and change the method signatures accordingly.

Reply:
Here I need file name only not file path. File path is providing absolute path if i am keeping it as string. For retrieving only the file name from absolute path i am making File from string path.

Reply:
@smahajan05 I understand that. What I am saying is that you don't need to create File object as such.
You are first creating path and then creating file from that and then just using the name of the file.
WIll this work?

Paths.get("whatever_str").getFileName()

I suggested this minor change only.